app/helpers/view_components/track/generic_nav.rb

```diff
@@ -7,14 +7,14 @@ def to_s
         tag.nav(class: klass) do
           tag.div(class: 'lg-container container') do
             safe_join([
-              link_to(
-                graphical_icon("arrow-left"),
-                Exercism::Routes.tracks_path,
-                class: "back",
-                'aria-label': "Back to all tracks"
-              ),
+              # link_to(
+              #   graphical_icon("arrow-left"),
+              #   Exercism::Routes.tracks_path,
+              #   class: "back",
+              #   'aria-label': "Back to all tracks"
+              # ),
```

Comment:
Does this need a TODO indicating why the code is commented?

Reply:
Done :)